base/src/main/java/me/lambdaurora/mcpatcherpatcher/mcpatcher/MCPatcherParser.java

```diff
@@ -158,4 +158,26 @@ private static int parseInt(@NotNull String input, int defaultValue)
 
         return result;
     }
+
+    public static Number toTickTime(@NotNull String time)
+    {
+        String[] parts = time.split(":");
+        if (parts.length != 2)
+            return null;
+        int h = Integer.parseInt(parts[0]);
+        int m = Integer.parseInt(parts[1]);
+        int minutes = (h + (h >= 6 ? -6 : 18)) * 60 + m;
+        return minutes * 1000 / 60;
+    }
+
+    public static Number normalizeTickTime(int tickTime)
```

Comment:
```suggestion
    public static int normalizeTickTime(int tickTime)
```
I don't think it should return `Number`, it's always an int.